src/main/java/frc/team2412/robot/Hardware.java

```diff
@@ -163,6 +162,9 @@ public Hardware() {
         }
         if (SHOOTER_VISION_ENABLED) {
             limelight = new PhotonCamera(LIMELIGHT);
+            if (navX == null) {
+                navX = new NavX(GYRO_PORT);
+            }
```

Comment:
I believe we just moved to using a Pigeon, so will want to merge in that change when it's ready

Reply:
#93 also made some changes to vision we should incorporate too.

Reply:
Vision changes from #93 have been incorporated, but the Pigeon change hasn't been finished yet. Should we continue to delay merging all of these changes in because of that, or should we merge these in now and change to a Pigeon when that's ready?